pubsub/google/cloud/pubsub_v1/subscriber/policy/thread.py

```diff
@@ -167,7 +191,7 @@ def on_response(self, response):
         For each message, schedule a callback with the executor.
         """
         for msg in response.received_messages:
-            _LOGGER.debug('New message received from Pub/Sub: %r', msg)
+            _LOGGER.debug('New message received from Pub/Sub:\n%r', msg)
```

Comment:
@lukesneeringer Is there a reason you used `%r` instead of `%s` here? There is actually no distinction for protobuf types:

```python
>>> from google.cloud.pubsub_v1 import types
>>> msg = types.PubsubMessage(data=b'a', message_id=b'123f')
>>> str(msg)
'data: "a"\nmessage_id: "123f"\n'
>>> repr(msg)
'data: "a"\nmessage_id: "123f"\n'
>>> msg
data: "a"
message_id: "123f"

>>>
```

Reply:
Because in principle, `%r` is what I would want if there _was_ a distinction.

Reply:
@lukesneeringer So then I should change every other line to do the same? (i.e. `%r` rather than  `%s`, `{!r}` rather than `{}`)

Reply:
I went ahead and did it.